README.md

```diff
@@ -1,34 +1,85 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+![cover](https://user-images.githubusercontent.com/33742791/142388797-fa14bcfa-b01e-45ab-a271-bfdeeb26285f.png)
 
-## Getting Started
+<!--- insert badge here --->
 
-First, run the development server:
+[CU Get Reg](https://cugetreg.com) เว็บไซต์ที่จะช่วยอำนวยความสะดวกให้กับนิสิตจุฬาลงกรณ์มหาวิทยาลัยทุกคนในการค้นหาวิชาเรียนและวางแผนจัดตารางเรียนล่วงหน้าก่อนการลงทะเบียนจริง พัฒนาโดยชมรม [Thinc.](https://thinc.in.th)
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+## สารบัญ
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+- [จุดเริ่มต้นของ CU Get Reg](#beginning-of-cu-get-reg)
+- [โครงการที่เกี่ยวข้อง](#related-repositories)
```

Comment:
Inconsistent header (`โครงการ` กับ `โปรเจค`) P.S. Why not just say related repos (`Repository ที่เกี่ยวข้อง`)? These are the same project after all